### ISSUE TITLE
Problem to run "run_lstm_RGB.sh" on lisa-anna-public

### ISSUE THREAD

**Issue by haohao900618** [LisaAnne/lisa-caffe-public#6]
when I replicate the activity recognition experiments in the paper "Long-term Recurrent Convolutional Networks for Visual Recognition and Description", I download the repository from LisaAnne/lisa-caffe-public. I have only changed the path of UCF-101 frames to a specific path, such as, RGB_frames = '~/frames' in sequence_input_layer.py file, but I got a strange promblem that the label of frames is decimals number only on test phase, and I got the normal label on train phase.
some results when run "run_lstm_RGB.sh"  as follow:
......
I1210 09:15:40.268235 25677 solver.cpp:340] Test net output #42: reshape-label = 52.52
I1210 09:15:40.268242 25677 solver.cpp:340] Test net output #43: reshape-label = 46.68
I1210 09:15:40.268249 25677 solver.cpp:340] Test net output #44: reshape-label = 53.31
I1210 09:15:40.268257 25677 solver.cpp:340] Test net output #45: reshape-label = 52.52
I1210 09:15:40.268265 25677 solver.cpp:340] Test net output #46: reshape-label = 46.68
I1210 09:15:40.268272 25677 solver.cpp:340] Test net output #47: reshape-label = 53.31
I1210 09:15:40.268280 25677 solver.cpp:340] Test net output #48: reshape-label = 52.52
I1210 09:15:40.268287 25677 solver.cpp:340] Test net output #49: reshape-label = 46.68
I1210 09:15:42.790904 25677 solver.cpp:214] Iteration 0, loss = 4.61574
I1210 09:15:42.790948 25677 solver.cpp:229] Train net output #0: accuracy = 0.00260417
I1210 09:15:42.790963 25677 solver.cpp:229] Train net output #1: loss = 4.61574 (* 1 = 4.61574 loss)
I1210 09:15:42.790972 25677 solver.cpp:229] Train net output #2: reshape-label = 89
I1210 09:15:42.790982 25677 solver.cpp:229] Train net output #3: reshape-label = 52
I1210 09:15:42.790988 25677 solver.cpp:229] Train net output #4: reshape-label = 74
I1210 09:15:42.790995 25677 solver.cpp:229] Train net output #5: reshape-label = 36
I1210 09:15:42.791002 25677 solver.cpp:229] Train net output #6: reshape-label = 21
I1210 09:15:42.791009 25677 solver.cpp:229] Train net output #7: reshape-label = 49
I1210 09:15:42.791016 25677 solver.cpp:229] Train net output #8: reshape-label = 77
I1210 09:15:42.791023 25677 solver.cpp:229] Train net output #9: reshape-label = 17
......
Is there anyone who has encountered the problem like me?
thank for your help!

**Comment by LisaAnne** [comment]
Hi - I am also seeing this and will work on figuring out what is going on!  I did recently use this code to train some new models, and replicated the results from the paper.

**Comment by haohao900618** [comment]
Oh. Thank for your attention. I have worked on it for half a month, but I can not solve the problem.

**Comment by LisaAnne** [comment]
Hi - I went to debug this and could not replicate your error.  You can see how I tried to replicate your output in the new branch: "lstm_video_deploy_labelDebug".  My logfile is here: https://github.com/LisaAnne/lisa-caffe-public/blob/lstm_video_deploy_labelDebug/examples/LRCN_activity_recognition/debugLabels.out.
When printing labels after the reshape-label layer I see:

#################################################################
I1214 10:12:24.970284 24007 solver.cpp:340]     Test net output #38: reshape-label-print-test = 45
I1214 10:12:24.970293 24007 solver.cpp:340]     Test net output #39: reshape-label-print-test = 52
I1214 10:12:24.970300 24007 solver.cpp:340]     Test net output #40: reshape-label-print-test = 80
I1214 10:12:24.970309 24007 solver.cpp:340]     Test net output #41: reshape-label-print-test = 45
I1214 10:12:24.970315 24007 solver.cpp:340]     Test net output #42: reshape-label-print-test = 52
I1214 10:12:24.970322 24007 solver.cpp:340]     Test net output #43: reshape-label-print-test = 80
I1214 10:12:24.970330 24007 solver.cpp:340]     Test net output #44: reshape-label-print-test = 45
I1214 10:12:24.970340 24007 solver.cpp:340]     Test net output #45: reshape-label-print-test = 52
I1214 10:12:24.970347 24007 solver.cpp:340]     Test net output #46: reshape-label-print-test = 80
I1214 10:12:24.970355 24007 solver.cpp:340]     Test net output #47: reshape-label-print-test = 45
I1214 10:12:24.970362 24007 solver.cpp:340]     Test net output #48: reshape-label-print-test = 52
I1214 10:12:24.970369 24007 solver.cpp:340]     Test net output #49: reshape-label-print-test = 80
I1214 10:12:27.291668 24007 solver.cpp:214] Iteration 0, loss = 4.61408
I1214 10:12:27.291712 24007 solver.cpp:229]     Train net output #0: accuracy = 0.0208333
I1214 10:12:27.291724 24007 solver.cpp:229]     Train net output #1: loss = 4.61408 (* 1 = 4.61408 loss)
I1214 10:12:27.291731 24007 solver.cpp:229]     Train net output #2: reshape-label-print-train = 89
I1214 10:12:27.291743 24007 solver.cpp:229]     Train net output #3: reshape-label-print-train = 52
I1214 10:12:27.291749 24007 solver.cpp:229]     Train net output #4: reshape-label-print-train = 74
I1214 10:12:27.291754 24007 solver.cpp:229]     Train net output #5: reshape-label-print-train = 36
I1214 10:12:27.291767 24007 solver.cpp:229]     Train net output #6: reshape-label-print-train = 21
I1214 10:12:27.291774 24007 solver.cpp:229]     Train net output #7: reshape-label-print-train = 49
I1214 10:12:27.291779 24007 solver.cpp:229]     Train net output #8: reshape-label-print-train = 77
I1214 10:12:27.291786 24007 solver.cpp:229]     Train net output #9: reshape-label-print-train = 17
I1214 10:12:27.291792 24007 solver.cpp:229]     Train net output #10: reshape-label-print-train = 3
I1214 10:12:27.291798 24007 solver.cpp:229]     Train net output #11: reshape-label-print-train = 49
I1214 10:12:27.291805 24007 solver.cpp:229]     Train net output #12: reshape-label-print-train = 60
I1214 10:12:27.291810 24007 solver.cpp:229]     Train net output #13: reshape-label-print-train = 38
I1214 10:12:27.291816 24007 solver.cpp:229]     Train net output #14: reshape-label-print-train = 59
I1214 10:12:27.291822 24007 solver.cpp:229]     Train net output #15: reshape-label-print-train = 45
I1214 10:12:27.291832 24007 solver.cpp:229]     Train net output #16: reshape-label-print-train = 56
I1214 10:12:27.291838 24007 solver.cpp:229]     Train net output #17: reshape-label-print-train = 55
I1214 10:12:27.291844 24007 solver.cpp:229]     Train net output #18: reshape-label-print-train = 98
I1214 10:12:27.291851 24007 solver.cpp:229]     Train net output #19: reshape-label-print-train = 77
I1214 10:12:27.291857 24007 solver.cpp:229]     Train net output #20: reshape-label-print-train = 76
I1214 10:12:27.291862 24007 solver.cpp:229]     Train net output #21: reshape-label-print-train = 37
I1214 10:12:27.291868 24007 solver.cpp:229]     Train net output #22: reshape-label-print-train = 80

**Comment by haohao900618** [comment]
Dear Lisa Anna：
   The problem still exist in my system. you have said before that you have saw this problem like mine. Could you tell me that how did you solve it?
thanks very much for you.
